### PR TITLE
[CHANGE] Unify style of profile editor and display

### PR DESCRIPTION
**Pull request by lina-cat**
Uses the same visual style and fonts for profile and profile editor.
This allows for the creation of ASCII art, visually consistent indentation ect.

### CODE REVIEW

File: pandora-client-web/src/components/profileScreens/profileScreens.scss
Comment (ClaudiaMia) [inline pandora-client-web/src/components/profileScreens/profileScreens.scss:60]: I think this is absolutely fine as a quick improvement of this feature.

I just wondered about what Ellie said, that typically there would also be a banner above the input field or even higher on the page in yellow or red that warns about you being in edit mode.

Because if you write for half an hour on your profile and then make the silly mistake of assuming that it autosaved and you can press back or otherwise navigate away, you will be devastated!

A potentially most simple solution to deal with that is to give the read mode the same background color as the chat, so:

```
		color: #000;
		background-color: $grey-lightest;
```

and then in the `.profileEdit`

```
		border: 2px dashed $red-dark;
		color: #000;
		background-color: $grey-lightest;
```

But those are just some thoughts of mine. I just feel in the long term we need to address the risk of losing your changes even further.

**Review by ClaudiaMia — APPROVED** [review]
Tested that it works well.

Needs 2nd review.

**Review by Sekkmer — APPROVED** [review]
looks good to me too

@ClaudiaMia feel free to merge this after closing the comment

File: pandora-client-web/src/components/profileScreens/profileScreens.scss
Comment (lina-cat) [reply pandora-client-web/src/components/profileScreens/profileScreens.scss:60]: for future solutions of this, I would anyway recommend to save the "in progress" edit at least in the browser so you can recover it even after you close the window (plus a button to reset to current profile) but that would really be out of scope here

Comment (lina-cat) [reply pandora-client-web/src/components/profileScreens/profileScreens.scss:60]: pushed changes, so another quick review would be appreciated before merge

**Pull request merged**
